src/main/js/components/CnesReportApp.js

```diff
@@ -121,8 +146,60 @@ export default class CnesReportApp extends React.PureComponent {
                                 onChange={this.onChangeAuthor} />
                             <input type="hidden" name="token" id="token_cnesreport" defaultValue={this.state.token} />
                         </div>
+                        <div>
+                            <input id="enableDocxHidden" type="hidden" value="false" name="enableDocx" disabled={this.state.enableDocx}/>

```

Comment:
We should insert some comments to explain why we add hidden fields in the form, related to how the plugin API works, and so on.